src/main/java/org/traccar/notificators/NotificatorViber.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2019 - 2023 Anton Tananaev (anton@traccar.org)
+ * Copyright 2021 Rafael Miquelino (rafaelmiquelino@gmail.com)
```

Comment:
Fix this. You can either put your name/email or keep mine, but the dates should be right.